Source/DataAccessLayer/Utility.cst

```diff
@@ -307,7 +307,7 @@ namespace <%=DALNameSpace%>
 
         #region SqlInjection
         private static readonly System.Text.RegularExpressions.Regex regSystemThreats =
-                new System.Text.RegularExpressions.Regex(@"\s?;\s?|\s?drop\s|\s?grant\s|^'|\s?--|\s?union\s|\s?delete\s|\s?truncate\s|\s?sysobjects\s?|\s?xp_.*?|\s?syslogins\s?|\s?sysremote\s?|\s?sysusers\s?|\s?sysxlogins\s?|\s?sysdatabases\s?|\s?aspnet_.*?|\s?exec\s?",
+                new System.Text.RegularExpressions.Regex(@"\s?;\s?|\bdrop\b|\bgrant\b|^'|\s?--|\bunion\b|\bdelete\b|\btruncate\b|\bsysobjects\b|\s?xp_.*?|\bsyslogins\b|\bsysremote\b|\bsysusers\b|\bsysxlogins\b|\bsysdatabases\b|\baspnet_.*?|\bexec\b",

```

Comment:
What does `\b` do?

Reply:
word break.  It includes spaces but also handles things like returns.